EnglishFilter/sections/specific.txt

```diff
@@ -11748,6 +11748,9 @@ adultlingeriepics.com,adultlingeriepics.com,adultspankingtgp.com,allfeetporn.com
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/73030
+!+ PLATFORM (ios)
+nytimes.com##div[id^="ad-"]
```

Comment:
Is it blocked in Safari for Mac and by AdGuard Content Blocker?

Reply:
@Alex-302,
I wasn't able to reproduce these leftovers in Samsung browser at all.
I reproduced them in Yandex browser on Android and Countent blocker blocked them.